src/abstractprocess.h

```diff
@@ -41,19 +41,14 @@ class AbstractProcess
     virtual ~AbstractProcess() {}
 
 public:
-    QString app() const;
-
-    void setInputFile(const QString &file);
-    QString inputFile() const;
-
     virtual void execute() = 0;
     QProcess::ProcessState state() const;
 
     FileId groupId() const;
     void setGroupId(const FileId &groupId);
 
 signals:
-    void finished(AbstractProcess *process, int exitCode);
+    void finished(FileId origin, int exitCode);
```

Comment:
@rgoltermann Can we get rid of the `FileId` stuff in the AbstractProcess? Probably, we can move all the `ProjectGroupNode` related stuff in `MainWindow::execute()` and `MainWindow::postGamsRun()` to the `ProjectGroupNode`. We should have a look...

Reply:
we won't change this now.